src/_support/Page/Acceptance/Administrator/MenuItem.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+    /**
+     * @package     Joomla.Test
+     * @subpackage  AcceptanceTester.Page
+     *
+     * @copyright   Copyright (C) 2005 - 2019 Open Source Matters, Inc. All rights reserved.
+     * @license     GNU General Public License version 2 or later; see LICENSE.txt
+     */
+
+    namespace Page\Acceptance\Administrator;
+
+    class MenuItem
+    {
+        // include url of current page
+        public static $url = "administrator/index.php?option=com_menus&view=items";
+
+        /**
+         * New User Button
+         *
+         * @var    string
+         * @since  3.7.3
+         */
+        public static $newButton = ['id' => 'toolbar-new'];
+
+        /**
+         * Menu Item Title
+         *
+         * @var    string
+         * @since  3.7.3
+         */
+        public static $menuItemTitle = ['id' => 'jform_title'];
+
+        /**
+         * Menu Item Alias
+         *
+         * @var    string
+         * @since  3.7.3
+         */
+        public static $menuItemAlias = ['id' => 'jform_alias'];
+
+        /**
+         * Menu Item Save
+         *
+         * @var    string
+         * @since  3.7.3
+         */
+        public static  $saveButton = ['id' => 'toolbar-apply'];
+
+        /**
+         * Page title of the user manager listing page.
+         *
+         * @var    string
+         * @since  3.7.3
+         */
+        public static $pageTitleText = "Menus";
+
+
+        /**
+         * drop down menu
+         *
+         * @var    string
+         * @since  3.7.3
+         */
+        public static $menuDropDown = ['xpath' => '//select[@id="jform_menutype"]'];
```

Comment:
Change Locators to strings instead of Arrays, with Codeception `2.4.x`  assert functions only take strings as Arguments, it is better to change these in Page Objects, I will have another task for you which will be just to update all locators in page classes
These are the changes you will do 
`['xpath' => "//somepath"]  => "//something"`
`['id' => "someid"] => "#someid"`
`['Class' => "someclass"] => ".someclass"`

Reply:
You can skip this comment for now, I will handle it later.